### PR TITLE
added a post on deploying rails to docker

### PR DESCRIPTION
**Pull request by anujmiddha**

### CODE REVIEW

File: source/_posts/rails-docker.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+---
+title: Deploying a Rails app to a Docker container
```
Comment (Rajeev111974) [inline source/_posts/rails-docker.md:2]: Title tag character should be 50-60 character

File: source/_posts/rails-docker.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
+---
```
Comment (Rajeev111974) [inline source/_posts/rails-docker.md:1]: in content there should be link with previous content

Comment (Rajeev111974) [inline source/_posts/rails-docker.md:1]: keyword should be linked with previous nayan blog if possible

Comment (Rajeev111974) [inline source/_posts/rails-docker.md:1]: Image should be optimized with Alt text

Comment (Rajeev111974) [inline source/_posts/rails-docker.md:1]: 1-2 long   tail keyword should be there if possible

**Comment by Rajeev111974** [comment]
HI Anuj,

 

We have reviewed the blog and find some changes is there, please check and do the thing needful.

 

With Regards

Rajeev Kumar

Operations Manager, 

Nayan India Science & Technologies (P) Ltd.

+919958083303

rajeev@nayantech.com

 

This e-mail and any attachments contain information from NAYAN Companies and are intended solely for the use of the named recipient or recipients. This e-mail may contain confidential and proprietary information. Any dissemination of this e-mail by anyone other than an intended recipient is strictly prohibited. If you are not a named recipient, you are prohibited from any further viewing of the e-mail or any attachments or from making any use of the e-mail or attachments. If you believe you have received this e-mail in error, notify the sender immediately and permanently delete the e-mail, any attachments, and all copies thereof from any drives or storage media and destroy any printouts of the e-mail or attachments.

File: source/_posts/rails-docker.md (hunk 1 above)
Comment (anujmiddha) [reply source/_posts/rails-docker.md]: Done

File: source/_posts/rails-docker.md (hunk 2 above)
Comment (anujmiddha) [reply source/_posts/rails-docker.md:1]: This is not linked to our previous posts. So can't add a link.

Comment (anujmiddha) [reply source/_posts/rails-docker.md:1]: Done.

Comment (anujmiddha) [reply source/_posts/rails-docker.md:1]: We have two keywords.

Comment (anujmiddha) [reply source/_posts/rails-docker.md:1]: Its not linked with previous posts.

**Comment by anujmiddha** [comment]
@Rajeev111974 Please check.

Thanks.

**Pull request merged**